examples/kitchen-sink/__test__/integration.test.js

```diff
@@ -21,7 +21,7 @@ describe('examples/kitchen-sink/integration', () => {
     expect(response.text).toContain('data-trackable="logo" href="/"')
   })
 
-  it('renders the header top components; search, menu and myFT', () => {
+  it('renders the header top components; search and menu', () => {
```

Comment:
**Reviewer notes:** myFT was actually never tested in this test, it also no longer made sense in this test because its now dependant on users logged in state. So this is just a test description tidy up